File: src/Taller1/ej2/Counter.java
package Taller1.ej2;

public class Counter{
    private int value;
    public Counter(){
        value = 0;
    }
    public int getValue(){
        return this.value;
    }
    public void increment(){
        this.value++;
    }
    public void decrement(){
        this.value--;
    }
    public void reset(){
        this.value = 0;
    }
}

File: src/Taller10/ej1/counter.rb
# frozen_string_literal: true

class Counter
    attr_reader :count
    def initialize
        @count = 0
    end

    def increment
        @count += 1
    end

    def decrement
        @count -= 1
    end

    def count=(value)
        @count = value
    end

    def to_s
        "#{@count}"
    end
end
